webapp/src/test/java/org/apache/atlas/examples/QuickStartV2IT.java

```diff
@@ -155,6 +157,25 @@ private void verifySimpleTableAttributes(AtlasEntity table) {
         assertEquals("sales fact table", tableAttributes.get("description"));
     }
 
+    private void verifyQualifiedNames(AtlasEntity table) {
+        Map<String, Object> tableAttributes = table.getRelationshipAttributes();
+        Map                 dbFromTable     = (Map) tableAttributes.get("db");
+        String              tableName       = (String) tableAttributes.get("name");
+        String              dbName          = (String) dbFromTable.get("name");
```

Comment:
'dbFromTable' is an AtlasObjectId and wouldn't have all attributes of the DB entity, hence dbName should be null here. Can you please confirm if this test passes successfully?

Changes in QuickStartV2.java look good, though.